FreeIT/src/lesson5/Tictactoe.java

```diff
@@ -20,11 +20,11 @@ public static void main(String[] args) {
 		hodX();
 		for (int i = 1; i <= 4; i++) {
```

Comment:
can't find this row:
 static String game[][] = { { "1", "2", "3" }, { "4", "5", "6" }, { "7", "8", "9" } };
please pay attention to the hint - it suggests to use java style array declaration:
static String[][] game = { { "1", "2", "3" }, { "4", "5", "6" }, { "7", "8", "9" } };
Good code style to name static variables with apper case: static String[][] GAME.

static int count = 0; - not sure that it should be global static variable. I see that you use check method. it could return count variable for example. But better to return true/false from check as end of game parameter.